android/build.gradle

```diff
@@ -4,19 +4,19 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.3.1'
+        classpath 'com.android.tools.build:gradle:2.3.3'
     }
 }
 
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.1"
+    compileSdkVersion 26
+    buildToolsVersion '26.0.1'
```

Comment:
I'm not so sure we should excel passed the versions based on React Native.

Link: https://github.com/facebook/react-native/blob/1e8f3b11027fe0a7514b4fc97d0798d3c64bc895/ReactAndroid/build.gradle#L234-L235

Whenever I've tried to upgrade beyond what's officially supported by RN it's caused issues.  Not so sure we can bump these numbers until core chooses to do so officially.  Otherwise, we'll be confusing a lot of poor people just trying to add a gradient.

Reply:
Thank you for reviewing my PR. I completely understand your concern and I'll look for more info and see what kind of issues we can have on bumping the version.

I'll let you know if I can get more useful info about that.

Reply:
@GantMan There are some examples of other projects using a higher version:

https://github.com/react-native-community/react-native-video/blob/master/android/build.gradle

I have been using that and this project in production and everything seems stable. 

Now I am updating my project to a higher buildToolVersion and this PR would help me out a-lot as I am sure it will help others as well. leaving it at 23 is very limiting in my opinion.

Reply:
@casajavi and @GantMan, actually there are more examples from [*react-community*](https://github.com/react-community):

- [`react-native-languages`](https://github.com/react-community/react-native-languages/blob/7587c40111d4862bd7cf4ea3c6a46863a765e34f/android/build.gradle)
- [`react-native-image-picker`](https://github.com/react-community/react-native-image-picker/blob/1efe791f3aa6c9bab8caef60214e02aba5417eae/android/build.gradle)